Podfile

```diff
@@ -171,8 +171,8 @@ target 'WordPress' do
     ##
     pod 'Charts', '~> 3.2.2'
     pod 'Gifu', '3.2.0'
-    pod 'AppCenter', '2.5.1', :configurations => ['Release-Internal', 'Release-Alpha']
-    pod 'AppCenter/Distribute', '2.5.1', :configurations => ['Release-Internal', 'Release-Alpha']
+    pod 'AppCenter', '4.1.1', :configurations => ['Release-Internal', 'Release-Alpha']
+    pod 'AppCenter/Distribute', '4.1.1', :configurations => ['Release-Internal', 'Release-Alpha']
```

Comment:
This is quite the version bump.

Was it due to the Fastlane plugin update? If so, weird that they did it on a patch version bump .

Reply:
I did this by hand to get it up to date 😃